terminus/builders/osm_city_builder.py

```diff
@@ -128,10 +123,11 @@ def _create_roads(self, city):
                     # Check if coord is inside bounding box
                     ref_lat = self.osm_coords[ref]['lat']
                     ref_lon = self.osm_coords[ref]['lon']
-                    coord = self._translate_coords(ref_lat, ref_lon)
+                    ref_point = LatLon(ref_lat, ref_lon)
```

Comment:
Try to be consistent with the names, otherwise things look quite messy. The variable name is `ref_point`, but actually holds a `LatLon`, which is then converted to an actual point, but it is named `coord`  :confused: